docs/tutorials/blog.md

```diff
@@ -755,7 +760,7 @@ export default function NewPost() {
 
 If you love HTML like us, you should be getting pretty excited. If you've been doing a lot of `<form onSubmit>` and `<button onClick>` you're about to have your mind blown by HTML.
 
-All you really need for a feature like this is a form to get data from the user and a backend action to handle it. And in Remix, that's all you have to.
+All you really need for a feature like this is a form to get data from the user and a backend action to handle it. And in Remix, that's all you have to do, too.
```

Comment:
Not sure if this is me not being a native speaker, but the sentence

> And in Remix, that's all you have to.

is missing a "do", isn't it?